src/modules/bookmark/utils.js

```diff
@@ -26,17 +25,25 @@ export const validateBookmarks = (data) => {
 export const getIndexOfBookmark = (bookmarks, { address, token = tokenMap.LSK.key }) =>
   bookmarks[token].findIndex((bookmark) => bookmark.address === address);
 
+export const getIndexOfLabel = (bookmarks, { label, token = tokenMap.LSK.key }) =>
+  bookmarks[token].findIndex((item) => item.title === label);
+
```

Comment:
The usage of `token` is wrong? `tokenMap.LSK.key` this is hardcoding, also we never need to store bookmark specific to any token? its a old code..